core/templates/dev/head/AppInit.ts

```diff
@@ -14,7 +14,7 @@
 
 var oppia = angular.module(
   'oppia', [
-    'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
+    'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
```

Comment:
Should we move some of this up?

Reply:
Yeah I can move this up for readability.

Reply:
Moved some of this up, should be done.